src/components/PopupWithForm.js

```diff
@@ -14,13 +15,19 @@ export default class PopupWithForm extends Popup {
     return inputValues;
   }
 
+  open() {
+    this._submitButton.textContent = "Save";
```

Comment:
Needs correction: please don't change the button text here. You should create a separate method for rendering loading and start the loading right before the api request and end the loading after in `finally` block